intranet/static/js/bus.js

```diff
@@ -1,707 +1,715 @@
 /* globals Messenger */
 var bus = {};
+var now = new Date();
+afterNoon = now.getHours() > parseInt("{{ changeover_time }}", 10);
```

Comment:
Honestly, the changes here are so large that I'd rather have them in separate files -- as it stands I can't even tell what's what to review it. Maybe keep the code to open/reopen the websocket in a `bus-shared.js`, and move everything else to `bus-morning.js` and `bus-afternoon.js`?

I also don't like the fact that `afterNoon` here is set based on the current time. Users should be able to see either the afternoon or morning bus page specifically, and the auto-switching functionality should only be on the main `/bus` URL. This would make future testing MUCH easier.

Reply:
The issue I see with allowing users to see either is that they use the same bus objects, so if an admin were to make a change in the afternoon, it would make a change in the morning page that might confuse users. I think I would also need to create a separate web socket, which is going to involve lots of repeated code. What if I kept the switchover, but allowed a static version of the opposite to still be displayed. So after 12, the morning bus page could be shown, but wouldn't be connected to a web socket. This would solve everything except the refresh issue.

Reply:
Ah, I missed the issue with the same bus objects. What you could do is alter the consumer so that it takes a `morning`/`afternoon` parameter in the URL, and then pretends that there are no buses if that doesn't match the actual time. The logic would be a little tricky, but I'd rather do it that way.

Also, you can't embed variables in static files like this.